### ISSUE TITLE
Project Red - Transmission bunled cables integration

### ISSUE THREAD

**Issue by Commandcracker** [tweaked-programs/cccbridge#75]
**Description**
The RedRouter Block should integrate with Project Red bunled cables, like base CC-Tweaked.

**Comment by SammyForReal** [comment]
Yes, I totally agree. They lately reintroduced compatibility with bundled cables within the latest updates.

Gonna figure out how to do so tho.

**Comment by SammyForReal** [comment]
Since CC:T (finally) has it's own redstone block by now, I will probably mark the RedRouter as deprecated and stop working on features for it. It will remain in the mod however for backwards compatibility.
